app/views/mailing_list/steps/completed/shared/_get_an_adviser_high_commitment.html.erb

```diff
@@ -1,7 +1,7 @@
-<h3 class="heading-s">Get an adviser</h3>
-
-<p>
-  Maximise your chances of submitting a successful application by getting
-  <a href="/teacher-training-advisers">free support from a dedicated adviser</a>
-  with years of teaching experience.
-</p>
+<%= render CallsToAction::SimpleComponent.new(
+  title: "Maximise your chances of a successful application",
+  text: "Our expert advisers have years of teaching experience. They can answer any questions you have and support you along your journey to becoming a teacher. Chat by phone, text or email, as little or as often as you need.",
+  icon: "icon-person",
+  link_text: "Find out more about free advisers",
```

Comment:
This is quite long link text (esp on mobile) and I think `free advisers` sounds a little odd without `our` as a qualifier

We could definitely lose the `more` - could we include free elsewhere? Would it be too lost in the body?

```suggestion
  link_text: "Find out about advisers",
```